workflow

```diff
@@ -0,0 +1,44 @@
+If I'm submitting a proposal, the exact workflow goes like this:
```

Comment:
Did you mean to include this?

Reply:
whoops sorry, committed some notes by mistake.  removed.